docs/products/opensearch/howto/connect-with-python.rst

```diff
@@ -0,0 +1,45 @@
+Connect to OpenSearch cluster with Python
+=========================================
+
+You can interact with your cluster with the help of the `Python OpenSearch client <https://github.com/opensearch-project/opensearch-py>`_. 
+It provides an easy syntax to send commands to your OpenSearch cluster. Follow its ``README`` file for installation instructions.
```

Comment:
I'd not use the word "easy", since what is easy for us, can be difficult for others. 
either remove it, or replace with something more neutral, like "convenient"